expirationd.lua

```diff
@@ -102,6 +102,11 @@ local function default_do_worker_iteration(task)
     local checked_tuples_count = 0
     local vinyl_checked_tuples_count = 0
     if task.atomic_iteration then
+        -- Before starting the transaction,
```

Comment:
"Check before starting transaction, ..." or something similar a phase "Before starting the transaction, since the transaction can be long" without any verb doesn't make sense.

Reply:
You are right, fixed on "Check before starting transaction, ..."